### PR TITLE
Landing page UI

### PR DESCRIPTION
**Pull request by sharathnair9999**
Landing Page UI (Category list Pending)
Video Context
Lazy loading screen while page loads (open devtools, right click on reload page button, clear cache and hard reload to check)

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]
UI looks great
Liked the use of Suspense for SplashScreen while data is being fetched

File: src/pages/VideoListing/VideoListing.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+          <option value="">Sort By</option>
+          <option value="latest">Latest</option>
+          <option value="oldest">Oldest</option>
+        </select>
+        <ul className="categories-chip-container">
+          <li className="">Unboxing</li>
+          <li className="">Review</li>
+          <li className="">Breakdown</li>
+          <li className="">Giveaways</li>
+        </ul>
```
Comment (prachi-sahani) [inline src/pages/VideoListing/VideoListing.js:18]: remove empty classnames

File: src/contexts/Video-Context/video-context.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const [videosState, videosDispatch] = useReducer(
+    videosReducer,
+    initialVideosState
+  );
+
+  const getAllCategories = async () => {
+    try {
+      const {
+        data: { categories },
+      } = await callAPI("GET", "/api/categories");
```
Comment (prachi-sahani) [inline src/contexts/Video-Context/video-context.js:17]: You can use better naming convention for callAPI(some name that signifies which API is being called)

File: src/App.css [hunk 3]
```diff
@@ -31,8 +33,8 @@
 
 .main-container {
   flex-grow: 1;
-  margin-top: 2rem;
-  padding: 1.5rem 1rem 1rem 1rem;
+  margin-top: 3.5rem;
+  padding: 1rem 1rem 0rem 1rem;
```
Comment (prachi-sahani) [inline src/App.css:37]: you need not mention the unit for 0 padding

**Review by prachi-sahani — COMMENTED** [review]
- Overall the code is well structured and good to merge.
- got to learn about lazy loading in react
- suggestion - remove console.log

File: src/pages/VideoListing/VideoListing.js (hunk 1 above)
Comment (sharathnair9999) [reply src/pages/VideoListing/VideoListing.js:18]: ya those are dummy. will change it

File: src/contexts/Video-Context/video-context.js (hunk 2 above)
Comment (sharathnair9999) [reply src/contexts/Video-Context/video-context.js:17]: that is a utility function to write axios easily,  which i am in turn using in actual API function where my naming convention would be according to the functionality of API

File: src/App.css (hunk 3 above)
Comment (sharathnair9999) [reply src/App.css:37]: Thanks for mentioning. Will remove that.

File: src/contexts/Video-Context/video-context.js (hunk 2 above)
Comment (prachi-sahani) [reply src/contexts/Video-Context/video-context.js:17]: okay

**Pull request merged**